kcards/templates/kcards.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html PUBLIC "-//IETF//DTD HTML 2.0//EN">
```

Comment:
Removing this seems to have no effect as of yet. I have no idea what it does, copy-pasta'd from something I found online. Seems removable.